src/main/java/nl/knaw/dans/catalog/db/TarPartEntity.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2022 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package nl.knaw.dans.catalog.db;
+
+import lombok.*;
+import org.hibernate.Hibernate;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+@Table(name = "tar_parts")
+@Getter
+@Setter
+@ToString
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class TarPartEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "part_name")
+    private String partName;
+    @Column(name = "checksum_algorithm")
+    private String checksumAlgorithm;
+    @Column(name = "checksum_value")
+    private String checksumValue;
+    @ManyToOne
+    @JoinColumn(name = "tar_uuid")
+    private TarEntity tar;
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
No annotation for this?

Reply:
Yes, but they recommend against doing this for JPA entities. See https://dzone.com/articles/lombok-and-jpa-what-may-go-wrong for more information.

Reply:
I would be tempted to change in ignorance. Comment everywhere might also be overkill. Don't know the balance.

Reply:
I have a plugin for IntelliJ by the creators of that article that warns me about that. It should also be possible to create some kind of linting rule that detects both @Entity and @Data on a class and rejects it.